verilog/tools/kythe/verilog_kythe_extractor.cc

```diff
@@ -80,7 +85,20 @@ Output: Produces Indexing Facts for kythe (http://kythe.io).
 )");
   const auto args = verible::InitCommandLine(usage, &argc, &argv);
 
-  std::vector<verilog::kythe::IndexingFactNode> indexing_facts_trees;
+  // Set to hold the unique directories for where to look for files.
+  std::set<std::string> include_dir_paths;
+  const std::string include_dir_string = absl::GetFlag(FLAGS_include_dir_paths);
+  if (!include_dir_string.empty()) {
+    const std::vector<absl::string_view> paths =
+        absl::StrSplit(include_dir_string.c_str(), ',');
+
+    for (absl::string_view path : paths) {
+      include_dir_paths.insert(std::string(path));
+    }
+  }
+
+  // Add the directory of the file list to included files directories.
+  include_dir_paths.insert(std::string(verible::file::Dirname(args[1])));
```

Comment:
I don't think we want this behavior by default -- it makes an assumption that is hard to undo.  This is best controlled by the invoker, who can always do `-I $(dirname filelist.txt)`.